environment/human_rendering_env.py

```diff
@@ -24,72 +58,137 @@ def __init__(
-        self.target = Target(self.target_data, True)
+        """
+        Create target object for self (with sprite).
+
+        Use target data to create Target object.
+        """
+        self._target = Target(self._target_data, True)
 
 
     def _create_background(self)-> None:
-        self.background = pygame.image.load(
-            self.env_data["background"]["sprite"]
+        """
+        Create background object for self.
+
+        Use environment data to create background object.
+        """
+        self._background = pygame.image.load(
+            self._env_data["background"]["sprite"]
         )
-        self.background = pygame.transform.scale(
-            self.background,
-            self.env_data["window_dimensions"]
+        self._background = pygame.transform.scale(
+            self._background,
+            self._env_data["window_dimensions"]
         )
 
-    def render(self)-> None:
-        self.screen.blit(self.background, (0, 0))
+    def _render(self)-> None:
+        """
+        Render function for all of the graphical elements of the 
+        environment.
+        """
+        self.screen.blit(self._background, (0, 0))
 
-        self.screen.blit(self.floor.sprite, [0, self.floor.coll_elevation])
-        self.screen.blit(self.agent.rot_sprite, self.agent.rot_rect)
-        self.screen.blit(self.target.sprite, self.target.rect)
+        self.screen.blit(self._floor.sprite, [0, self._floor.coll_elevation])
+        self.screen.blit(self._agent.rot_sprite, self._agent.rot_rect)
+        self.screen.blit(self._target.sprite, self._target.rect)
         
         pygame.display.flip()
         
     def step(self, action: int)-> np.ndarray:
+        """
+        Step function for environment.
+
+        Performs action on self._agent and renders frame.
+
+        @params:
+            - action (int): one of:
+                * 0:  do nothing
+                * 1: adjust pitch upwards
+                * 2: adjust pitch downwards
+                * 3: increase throttle
+                * 4: decrease throttle
+                * 5: shoot a bullet
+        
+        @returns:
+            - np.ndarray with observation of resulting conditions
+        """
         step_info = super().step(action=action)
 
-        self.render()
+        self._render()
 
         return step_info
     
     def reset(self, seed: int=42)-> tuple[np.ndarray, dict]:
-        super().reset()
+        """
+        Reset environment.
+
+        Will create completely new agent and target.
+        Adds new page to the history dictionary
+        return initial state & info. Renders the initial frame
+
+        @params:
+            - seed (int): seed used to spawn in the agent and target.
+        
+        @returns:
+            - np.ndarray with initial state 
+            (see self._calculate_observation())
+            - dict with info, made for compatibility with Gym 
+            environment, but is always empty.
```

Comment:
```suggestion
            environment, but is always empty
```

Reply:
het is gewoon een zin, maar dict is een keyword die geen hoofdletter kan krijgen